apps/web/modules/ee/contacts/api/v2/management/contacts/lib/contact.ts

```
@@ -0,0 +1,154 @@
+    // Create the contact and attributes in a transaction
+    const result = await prisma.$transaction(async (tx) => {
+      // Create the contact
+      const newContact = await tx.contact.create({
+        data: {
+          environmentId,
+        },
+      });
+
+      // Create all attributes for the contact
+      const attributeData = Object.entries(attributes).map(([key, value]) => {
+        const attributeKey = existingAttributeKeys.find((ak) => ak.key === key)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)